### ISSUE TITLE
ZL/ZR for navigation sometimes don't work

### ISSUE THREAD

**Issue by jeremyhalin** [djfumberger/retromega#6]
Sometimes I have to press ZL or ZR twice to go left or right.
I wanted to fix this bug, but couldn't reproduce on Pegasus-frontend on Windows.
Seems to be an issue related to Android version and maybe specifically to RP2.

**Comment by djfumberger** [comment]
Yeah, I'd noticed the same thing on device and thought it was a bug in the theme I couldn't track down.

But put together a small test theme to replicate the issue, and I think as you say it's an issue with Pegasus on Android or RP2.

[Pegasus Key Test.zip](https://github.com/djfumberger/retromega/files/5893665/Pegasus.Key.Test.zip)

I've logged a bug here:

https://github.com/mmatyas/pegasus-frontend/issues/781
